packages/visualizations/src/components/MapPoi/Map.ts

```diff
@@ -30,22 +40,31 @@ const CURSOR = {
     DRAG: 'move',
 };
 
-const POPUP_FEATURE_STATE_KEY = 'popup-feature';
+const ACTIVE_FEATURE_RATIO_SIZE = 1.3;
+
+/** Sorts features in a layer by setting a sort key for a specific feature. */
+const sortLayerFeatures = (
+    map: maplibregl.Map,
```

Comment:
We might as well `import type { Map, …}` I think. We can alias map if need be. It's minor but could lead to think that other types are note from the same namespace.

Reply:
done https://github.com/opendatasoft/ods-dataviz-sdk/pull/209/commits/ef13c4ff4f24b9e80597d2a83bde82589d5bd483